src/index.js

```diff
@@ -10,6 +16,12 @@ export {default as Shaders} from './shaders';
 export {default as Scene} from './scene';
 export * from './media';
 
+import {PhiloGL} from './core';
+
+if (typeof window !== 'undefined') {
```

Comment:
Our objective's is to make sure that the lib now works without global variables - so it would be great if installing PhiloGL as a global variable was optional rather than the default. 

   window.exportPhiloGL = function() { window.PhiloGL = PhiloGL; };

On the other hand it would require modifying all the example scripts to call this function. 

Maybe we need two versions of index.js, and two build-targets - one that is generates a bundle for HTML page scripts, that includes babel-polyfill and exports PhiloGL by default, and another that is used by browserify/webpack based applications that can require/import the lib directly?